demo/src/components/Demo/index.js

```diff
@@ -68,6 +71,20 @@ class Demo extends Component {
     window.removeEventListener('resize', this._updateWindowWidth);
   }
 
+  handleChangeTheme(e) {
+    this.setState({
+      theme: e.target.value
+    });
+  }
+
+  getTheme() {
```

Comment:
to ease adding of future themes, let's put all the logic in a separate file in `src/config.js`:
``` js
import themeWybo from 'reapop-theme-wybo';
import themeBootstrap from 'reapop-theme-bootstrap';

const DEFAULT_THEME = 'wybo'

const THEMES = {
  wybo: {
    label: "Wybo",
    theme: themeWybo
  },
  bootstrap: {
    label: 'Bootstrap',
    theme: themeBootstrap
  }
}
```